include/api.php

```diff
@@ -4495,16 +4485,12 @@ function api_account_update_profile_image($type)
 
 	// check if specified profile id is valid
 	if ($profileid != 0) {
-		$r = q(
-			"SELECT `id` FROM `profile` WHERE `uid` = %d AND `id` = %d",
-			intval(api_user()),
-			intval($profileid)
-		);
+		$profile = dba::selectFirst('profile', ['is-default'], ['uid' => api_user(), 'id' => $profileid]);
```

Comment:
Code standards: While you're at it, can you please rename the `$profileid` variable to `$profile_id`?

Reply:
I promise I won’t ask you anything else.